plot.py

```diff
@@ -0,0 +1,88 @@
+from dash import Dash, html, dcc, callback, Output, Input
+import plotly.express as px
+import pandas as pd
+from data_logging.mongodb.mongo import MongoDB
+from data_logging.mongodb.collection import MongoCollection
+from config.app_config import AppConfig
+from datetime import date, timedelta
+
+HOW_MANY_DAYS=90
+
+app = Dash(__name__)
+server = app.server
+config = AppConfig()
+mongo = MongoDB(config)
+
+all_collections = [e.value for e in MongoCollection]
+
+app.layout = html.Div([
+    html.H1(children='Membrane Community Metrics', style={'textAlign':'center'}),
+    dcc.Dropdown(all_collections, value="GoogleUsersInTutorialPerDay", id="collection-selection", clearable=False),
+    dcc.Dropdown(id='categories-selection', value=["Total"], clearable=False),
+    dcc.Graph(id='graph-content')
+])
+
+def densify(df, collection):
+    match collection:
+        case MongoCollection.HexCumulativePackagesDownloads:
+            pass
+        case MongoCollection.DiscordMembersAtDay:
+            pass
+        case default:
+            df.set_index("date",drop=True,inplace=True)
+            df.index = pd.to_datetime(df.index)
+            idx = pd.date_range(date.today()-timedelta(days=HOW_MANY_DAYS), date.today())
+            #idx = pd.date_range(df.index.min(), df.index.max())
+            df=df.reindex(idx)
+            df[df[collection.get_value_field_name()].isnull()] = 0
+            df['date']=df.index
+    return df
+
+@callback(
+    Output('categories-selection', 'options'),
+    Output('categories-selection', 'value'),
+    Output('categories-selection', 'multi'),
+    Output('categories-selection', 'disabled'),
+    Input('collection-selection', 'value')
+)
+def update_checkbox(collection_str):
+    collection = MongoCollection(collection_str)
+    df = pd.DataFrame(mongo.get_collection(collection))
+    if collection.get_meta_field_name() in df:
+        options = ["Total"]+list(df[collection.get_meta_field_name()].unique())
+        value = [options[0]]
+        multi = True
+        disabled = False
+    else:
+        options = ["Value"]
+        value = "Value"
+        multi = False
+        disabled = True
+
+    return options, value, multi, disabled
+
+@callback(
+    Output('graph-content', 'figure'),
+    Input('categories-selection', 'value'),
+    Input('collection-selection', 'value')
+)
+def update_graph(categories, collection_str):
```

Comment:
I think that `collection" and 'category' names aren't the best choice here.
I would use something like:
1. `metric_type` for `collection`
2. `metric_subcategory` or `metric_variant` for `category`

 At first glance, one may think that `category` refers to `metric category` - things like `DiscordMembersPerDay` or `UsersPerTutorialPerDay`, when it refers to something like `tutorial name` in `UsersPerTutorialPerDay` metric/